umm-spec-lib/src/cmr/umm_spec/validation/platform.clj

```diff
@@ -34,24 +34,45 @@
    with the model number. So the short name is not specific enough to identify
    specific platform instances. Therefore we are adding Characteristic names and values
    so that specific aircraft tail numbers, or boat IDS, etc. can be specified with the
-   short name."
+   short name. The |-| string is also added as a delimiter between the fields. This is used
```

Comment:
We should use the unicode group separator character! I only say that half jokingly.

If there are characters like "<" or ">" for example that are invalid for these fields that would be fine with me too.